testing/references/kpis/cost_dict_MultiZone.csv

```diff
@@ -0,0 +1,3 @@
+keys,value
```

Comment:
Is it possible that this reference file is coming from previous checkouts of the repo? The "cost" references should start by either of the following labels: `cost_constant`, `cost_dynamic` or `cost_highly_dynamic`. If that's the case this file must be removed.

Reply:
This reference was new when I ran the tests locally.  I think it is from ``test_iterative_call``.

Reply:
True, forgot about that one.